pkgs/development/interpreters/python/cpython/default.nix

```diff
@@ -279,6 +279,13 @@ in with passthru; stdenv.mkDerivation {
     find $out -name "*.py" | ${pythonForBuildInterpreter} -OO -m compileall -q -f -x "lib2to3" -i -
     '' + optionalString stripBytecode ''
     find $out -type d -name __pycache__ -print0 | xargs -0 -I {} rm -rf "{}"
+    '' + ''
+    # *strip* shebang from libpython gdb script - it should be dual-syntax and
+    # interpretable by whatever python the gdb in question is using, which may
+    # not even match the major version of this python. doing this after the
+    # bytecode compilations for the same reason.
+    mkdir -p $out/usr/share/gdb
+    sed '/^#!/d' Tools/gdb/libpython.py > $out/usr/share/gdb/libpython.py
```

Comment:
Why not:

```suggestion
    sed '/^#!/d' Tools/gdb/libpython.py > $out/share/gdb/libpython.py
```

?

Reply:
Sure, yeah. Will squash together with same commit.